insights/parsers/softnet_stat.py

```diff
@@ -93,7 +93,8 @@ def is_packet_drops(self):
         Returns:
             (bool): It will return True if observed packet drops.
         """
-        for each_cpu in self.data.keys()[1:]:
-            if self.data[each_cpu]['packet_drops']:
+        for each_cpu in self.nstat_per_cpu:
+            value = each_cpu.get('packet_drops', '_')
```

Comment:
Here you should replace the '_' with None, or just leave it as `each_cpu.get('packet_drops')`